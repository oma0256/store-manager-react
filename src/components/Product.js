import React from 'react';
import Col from 'react-bootstrap/Col';
import Card from 'react-bootstrap/Card';
import Button from 'react-bootstrap/Button';
import Row from 'react-bootstrap/Row';

const Product = props => {
  const { name, price, quantity } = props;
  return (
    <Card body border="primary" className="mb-3">
      <Row>
        <Col lg={8} className="text-center text-lg-left">
          <Card.Title>{name}</Card.Title>
          <p>Price: UGX {price}</p>
          <p>QTY: {quantity}</p>
        </Col>
        <Col lg={4} className="text-center text-lg-left">
          <Button>Details</Button>
        </Col>
      </Row>
    </Card>
  );
};

export default Product;
