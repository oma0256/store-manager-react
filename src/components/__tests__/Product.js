import React from 'react';
import { shallow } from 'enzyme';
import Product from '../Product';

describe('Product', () => {
  const product = shallow(<Product />);

  it('renders correctly', () => {
    expect(product).toMatchSnapshot();
  });
});
