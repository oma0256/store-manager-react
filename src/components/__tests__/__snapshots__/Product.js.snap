// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product renders correctly 1`] = `ShallowWrapper {}`;
